services/app/src/components/prompt/WindowPrompt.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function promptInput() {
+  if (
+    localStorage.getItem('user') === '' ||
+    localStorage.getItem('user') === null
+  ) {
+    const person = prompt('Please enter your name');
+    if (person != null) {
+      window.localStorage.setItem('user', person);
+    }
+  }
+}
+
+function WindowPrompt() {
+  return (
+    <div className="windowPrompt">
+      <script src={promptInput()} />
```

Comment:
Not sure how this got left again 😅

Reply:
Forgot to delete the component :sweat_smile:

Reply:
Should be fixed